docs/user-guide/src/capm3/introduction.md

```diff
@@ -0,0 +1,82 @@
+
+## Development Environment
+
+There are multiple ways to setup a development environment:
+
+- [Using Tilt](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-development-environment)
+- [Other management cluster](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#development-using-Kind-or-Minikube)
+- See [metal3-dev-env](https://github.com/metal3-io/metal3-dev-env) for an
+  end-to-end development and test environment for
+  `cluster-api-provider-metal3` and
+  [baremetal-operator](https://github.com/metal3-io/baremetal-operator).
+
+## Getting involved and contributing
+
+Are you interested in contributing to cluster-api-provider-metal3? We, the maintainers and community, would love your suggestions, contributions, and help! Also, the maintainers can be contacted at any time to learn more about how to get involved.
+
+To set up your environment checkout the [development environment](#development-environment).
+
+In the interest of getting more new people involved, we tag issues with [good first issue](https://github.com/metal3-io/cluster-api-provider-metal3/labels/good%20first%20issue). These are typically issues that have smaller scope but are good ways to start to get acquainted with the codebase.
+
+We also encourage ALL active community participants to act as if they are maintainers, even if you don’t have "official" write permissions. This is a community effort, we are here to serve the Kubernetes community. If you have an active interest and you want to get involved, you have real power! Don’t assume that the only people who can get things done around here are the "maintainers".
+
+We also would love to add more “official” maintainers, so show us what you can do!
+
+This repository uses the Kubernetes bots. See a full list of the commands [here.](https://go.k8s.io/bot-commands) Note that some of them might not be implemented it in metal3-ci.
+
+## Community
+
+Community details can be found [here.](https://github.com/metal3-io/metal3-docs#community)
+
+## Github issues
+
+We use Github issues to keep track of bugs and feature requests.
+There are two different templates to help ensuring that relevant information is included.
+
+### Bugs
+
+If you think you have found a bug please follow the instructions below.
+
+- Please spend a small amount of time giving due diligence to the issue tracker. Your issue might be a duplicate.
+- Get the logs from the cluster controllers. Please paste this into your issue.
+- Open a [bug report](https://github.com/metal3-io/cluster-api-provider-metal3/issues/new?assignees=&labels=&template=bug_report.md).
+- Remember users might be searching for your issue in the future, so please give it a meaningful title to help others.
+- Feel free to reach out to the cluster-api or metal3 community on [kubernetes slack.](https://github.com/kubernetes/community/tree/master/communication#slack)
```

Comment:
cluster-api? this looks a bit misleading to me to refer to kubernetes slack without referring to channel name(s). Should we maybe leave this out since we do give community details earlier which already includes this information?